### DISCUSSION TITLE
#1356 is marked not planned. Can I pay to change that?

### DISCUSSION THREAD

**Discussion by tiltowait** [Pycord-Development/pycord#1531] [Q&A]
#1356 is a serious bug with paginators using custom views in page groups. It's marked as "not planned". I won't ask about the reasons for that; however, I *will* ask if there's any way my team can pay to get it fixed? We're more than happy to support the project, but this particular issue is a major stumbling block at the moment.

**Comment by Lulalaby** [comment]
I'm sorry ext.pages is on break rn for internal reasons.
We can't help with that.
Maybe contact krittick.
